state-chain/cf-integration-tests/src/network.rs

```diff
@@ -883,3 +888,38 @@ pub fn register_refund_addresses(account_id: &AccountId) {
 		));
 	}
 }
+
+pub fn apply_extrinsic_and_calculate_gas_fee(
```

Comment:
Not sure why this function is here - it doesn't have much to do with `network` and is only used by the fee scaling test.